docs/user_guide/dynamic_provisioning.rst

```diff
@@ -14,26 +14,23 @@ Sample user.yaml for ``--add_user``:
 
     name: new_user@nvidia.com
     org: nvidia
-    roles:
-    - super
+    role: project_admin
  
 Sample client.yaml for ``--add_client``:
 
 .. code-block:: yaml
 
     name: new-site
     org: nvidia
-    enable_byoc: true
     components:
-    resource_manager:    # This id is reserved by system.  Do not change it.
-    path: nvflare.app_common.resource_managers.list_resource_manager.ListResourceManager
-    args:
-        resources:
-        gpu: [0, 1, 2, 3]
-    resource_consumer:    # This id is reserved by system.  Do not change it.
-    path: nvflare.app_common.resource_consumers.gpu_resource_consumer.GPUResourceConsumer
-    args:
-        gpu_resource_key: gpu
+      resource_manager:    # This id is reserved by system.  Do not change it.
+        path: nvflare.app_common.resource_managers.gpu_resource_manager.GPUResourceManager
+        args:
+          num_of_gpus: 0
+          mem_per_gpu_in_GiB: 0
+      resource_consumer:    # This id is reserved by system.  Do not change it.
+        path: nvflare.app_common.resource_consumers.gpu_resource_consumer.GPUResourceConsumer
+        args:
```

Comment:
just a suggestion we could move these parts into files and use ..include to import those files.

Reply:
This is for each site already.  Users who would like to add a new site via dynamic provisioning has to provide one single yaml file to describe the information.